### DISCUSSION TITLE
docs/quickstarts/python

### DISCUSSION THREAD

**Discussion by giscus** [fl0zone/fl0-docs#14] [Comments]
# docs/quickstarts/python

Unfortunately, sometimes problems happen! This page outlines some of the most common errors and provides advice on solving them.

https://docs.fl0.com/docs/quickstarts/python

**Comment by nicolasAT08** [comment]
Hello!

I'm new to web development/deployment and currently deploying a FastAPI app using Alembic for DB management. However, after the deployment, the Postgres tables' creation based on alembic revision didn't work..... It would be awesome if you guys could help me solve this.

Regards.
